examples/app-with-on-behalf-of-auth/docker-compose.yaml

```diff
@@ -6,8 +6,7 @@ services:
     build:
       context: ./.config
       args:
-        # Points to an unreleased v10.1 version of Grafana
-        grafana_version: ${GRAFANA_VERSION:-10.0.3}
+        grafana_version: ${GRAFANA_VERSION:-10.2.0}
```

Comment:
Just realizing with this that: https://github.com/grafana/grafana/pull/76673 didn't make it in `10.2.0`